### PR TITLE
move styleguide method into homes controller

### PR DESCRIPTION
**Pull request by ilyakatz**
it's a bit misleading that the styleguide method is in application_controller since the actual call to kss/styleguide is in homes_controller. plus, it should be public

### CODE REVIEW

File: app/controllers/kss/home_controller.rb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 module Kss
   class HomeController < ApplicationController
+    helper_method :styleguide
+
+    def styleguide
```
Comment (dewski) [inline app/controllers/kss/home_controller.rb:5]: The styleguide method wouldn't be accessible if others added more controllers. It's best to keep application wide methods like this in ApplicationController.

**Comment by dewski** [comment]
Thanks for the pull request! Unfortunately, the only change I could see being mergable is changing the styleguide method from private to public. I'd like to keep this engine pretty flexible if the user pleases, but they'd have to duplicate the styleguide method in each controller if they did.

**Comment by ilyakatz** [comment]
maybe i misunderstood your intentions, is styleguide supposed to be accessible in multiple places? i thought it was only one place for css documentation and that's why proposed the change. could you give an example please, i may be using it incorrectly in my project as well

thanks

**Comment by dewski** [comment]
You're most definitely using it correctly in your project. I am just aware of a few companies using KSS Rails and having multiple controllers to separate multiple parts of a site. For example, KSS::SiteController has (forms, links, buttons), and they also have a KSS::SupportController with another set of styles. If the `styleguide` method moved from the ApplicationController it'd have to be redefined in each controller in that case.

**Comment by ilyakatz** [comment]
got it, in that case just private-public change would great.

**Comment by dewski** [comment]
If you could update the pull request with that change I'll merge it.

**Pull request closed without merging**